### PR TITLE
[ValloxMV] Added some channels

### PR DESCRIPTION
**Pull request by bjoernbrings**
Added further channels which where requested in [forum](https://community.openhab.org/t/new-binding-vallox-mv-ventilation-unit-series-beta/36735/76) (for which I had no usecase so far).

Signed-off-by: Björn Brings <bjoernbrings@web.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java [hunk 1]
```diff
@@ -95,25 +95,19 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     valloxSendSocket.request(channelUID, "5");
                     valloxSendSocket.request(null, null);
                 }
-            } else if (ValloxMVBindingConstants.CHANNEL_EXTR_FAN_BALANCE_BASE.equals(channelUID.getId())) {
+            } else if (ValloxMVBindingConstants.CHANNEL_EXTR_FAN_BALANCE_BASE.equals(channelUID.getId())
+                    || ValloxMVBindingConstants.CHANNEL_SUPP_FAN_BALANCE_BASE.equals(channelUID.getId())
+                    || ValloxMVBindingConstants.CHANNEL_HOME_SPEED_SETTING.equals(channelUID.getId())
+                    || ValloxMVBindingConstants.CHANNEL_AWAY_SPEED_SETTING.equals(channelUID.getId())
+                    || ValloxMVBindingConstants.CHANNEL_BOOST_SPEED_SETTING.equals(channelUID.getId())) {
```
Comment (wborn) [inline addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java]: You could also create some constant Set with all dimensionless QuantityType channel IDs and then use the set to check if it contains the channel ID here.

File: addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java [hunk 2]
```diff
@@ -95,25 +95,19 @@ public void handleCommand(ChannelUID channelUID, Command command) {
...
-            } else if (ValloxMVBindingConstants.CHANNEL_HOME_SPEED_SETTING.equals(channelUID.getId())) {
-                if (command instanceof QuantityType) {
-                    QuantityType<Dimensionless> quantity = (QuantityType<Dimensionless>) command;
-                    valloxSendSocket.request(channelUID, Integer.toString(quantity.intValue()));
-                    valloxSendSocket.request(null, null);
-                }
-            } else if (ValloxMVBindingConstants.CHANNEL_HOME_AIR_TEMP_TARGET.equals(channelUID.getId())) {
+            } else if (ValloxMVBindingConstants.CHANNEL_HOME_AIR_TEMP_TARGET.equals(channelUID.getId())
+                    || ValloxMVBindingConstants.CHANNEL_AWAY_AIR_TEMP_TARGET.equals(channelUID.getId())
+                    || ValloxMVBindingConstants.CHANNEL_BOOST_AIR_TEMP_TARGET.equals(channelUID.getId())) {
```
Comment (wborn) [inline addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java]: You could also create some constant Set with all temperature QuantityType channel IDs and then use the set to check if it contains the channel ID here.

File: addons/binding/org.openhab.binding.valloxmv/ESH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -25,7 +25,11 @@
 			<channel id="extrfanbalancebase" typeId="extrfanbalancebase" />
 			<channel id="suppfanbalancebase" typeId="suppfanbalancebase" />
 			<channel id="homespeedsetting" typeId="homespeedsetting" />
+            <channel id="awayspeedsetting" typeId="awayspeedsetting" />
+            <channel id="boostspeedsetting" typeId="boostspeedsetting" />
 			<channel id="homeairtemptarget" typeId="homeairtemptarget" />
+            <channel id="awayairtemptarget" typeId="awayairtemptarget" />
+            <channel id="boostairtemptarget" typeId="boostairtemptarget" />
```
Comment (wborn) [inline addons/binding/org.openhab.binding.valloxmv/ESH-INF/thing/thing-types.xml]: Please reformat this file so tabs are used for indentation.

File: addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java (hunk 1 above)
Comment (bjoernbrings) [reply addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java]: done

File: addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java (hunk 2 above)
Comment (bjoernbrings) [reply addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java]: done

File: addons/binding/org.openhab.binding.valloxmv/ESH-INF/thing/thing-types.xml (hunk 3 above)
Comment (bjoernbrings) [reply addons/binding/org.openhab.binding.valloxmv/ESH-INF/thing/thing-types.xml]: done

**Comment by bjoernbrings** [comment]
Hey Wouter,
thanks for your quick review.
Hope to have resolved everything the way you suggested.

**Comment by wborn** [comment]
Thanks for the update Björn!
Can you also add the new channels to the binding documentation?  (README.md)

**Comment by bjoernbrings** [comment]
Should have remembered to update the documentation as well. Adapted.

**Pull request merged**